tests/communications/test_websocket_actor.py

```diff
@@ -0,0 +1,539 @@
+import uuid
+import pytest
+import pandas as pd
+import shutil
+
+from giskard.datasets.base import Dataset
+from giskard.ml_worker import ml_worker, websocket
+from giskard.ml_worker.utils.file_utils import get_file_name
+from giskard.ml_worker.websocket import listener
+from giskard.ml_worker.websocket.action import MLWorkerAction
+from giskard.models.base.model import BaseModel
+from giskard.settings import settings
+from giskard import slicing_function, transformation_function
+
+
+from tests import utils
+
+
+NOT_USED_WEBSOCKET_ACTOR = [
+    MLWorkerAction.generateQueryBasedSlicingFunction,
```

Comment:
Should we just remove them from the code ?

Reply:
Was having the concern that the enum value of WebSocketAction is shifted across versions, which may confuse the ml workers.
But it should be fine if we always explicitly indicate the enum values.

Reply:
Let's leave it like this for now, and then we'll rework how we use the enums after